optics/llnlWolter.rml

```diff
@@ -0,0 +1,34 @@
+<xml>
+	<globals>
+		<constant name="pi" value="3.1415927" />
+	</globals>
+
+	<TRestAxionTrueWolterOptics name="xmm" verboseLevel="warning" >
+		<parameter name="opticsFile" value="LLNL.Wolter" />
+
+		<parameter name="mirrorLength" value="150" />
+
+		<TRestAxionOpticsMirror name="LLNL_bilayer">
+			<parameter name="mirrorType" value="Bilayer" />
+
+		    <parameter name="layerThicknessTop" value="10.125" />
+		    <parameter name="sigmaTop" value="0" />
+
+		    <parameter name="layerBottom" value="Pt" />
+		    <parameter name="layerThicknessBottom" value="12.3755" />
+		    <parameter name="sigmaBottom" value="0.4" />
+
+			<parameter name="substrate" value="SiO2" />
+		</TRestAxionOpticsMirror>
+
+		<TRestSpiderMask name="spider" verboseLevel="warning">
```

Comment:
Need to figure out how to make that into the 2 mm wide graphite spacers. they were in the center which is 90° from the y-axis (as it is on the x-axis) so I think putting the armsSeperationAngle at 90° (or 360°/4) would make sense. The width is here in degree but can probably also be put in mm somehow.